plugins/aws_ecr/lib/samson_aws_ecr/samson_plugin.rb

```diff
@@ -0,0 +1,21 @@
+require 'aws-sdk'
+
+module SamsonAwsEcr
+  class Engine < Rails::Engine
+    mattr_accessor :ecr_client do
+      Aws::ECR::Client.new(region: ENV['AWS_REGION'] || "us-west-2")
```

Comment:
I had a problem when setting up the tests. 
`AWS_REGION` is the default variable `aws-sdk` uses for selecting the region. But I couldn't find a good place to set this variable (because `mattr_accessor` initializes when this file is loaded). Do you have a preferred way for doing this kind of client configuration?